ResultadosDigitais/feature_flagger
# Add cleanup detached rake

## What

- Rake to allow users to clean-up more easily;
- Update README explaining how to clean up;

## Testing

1. Add a new rollout in `rollout.yml`;
2. Release the new rollout to an account;
3. Remove the rollout from `rollout.yml`.
4. Check detached feature keys in storage using `FeatureFlagger::Manager.detached_feature_keys`;

- [ ] Must return the new rollout key;

5. Run/Execute `feature_flagger:cleanup_removed_rollouts` rake;
6. Check detached feature keys in storage  again;

- [ ] Must return no keys;
